### PR TITLE
install requirements file for common python

### PR DESCRIPTION
**Pull request by generalova-kate**
### Details:
 - move ownership from opencv to openvino team for  /opt/intel/openvino/python/requirements.txt file

### Tickets:
 - 48358

### CODE REVIEW

**Review by artanokhov — APPROVED** [review]

File: inference-engine/ie_bridges/python/CMakeLists.txt [hunk 1]
```diff
@@ -76,6 +76,14 @@ endif()
 
 # install
 
+ie_cpack_add_component(${PYTHON_BRIDGE_CPACK_PATH})
+
+install(FILES requirements.txt
```
Comment (mshabunin) [inline inference-engine/ie_bridges/python/CMakeLists.txt:81]: So is this the same file as the one installed to the `PYTHON_VERSION` subdirectory (line 89)? (https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/ie_bridges/python/requirements.txt) Maybe we could remove one of them?

**Review by mshabunin — APPROVED** [review]

File: inference-engine/ie_bridges/python/CMakeLists.txt (hunk 1 above)
Comment (generalova-kate) [reply inference-engine/ie_bridges/python/CMakeLists.txt:81]: yes, we will do it in the next interaction

File: inference-engine/ie_bridges/python/CMakeLists.txt [hunk 2]
```diff
@@ -76,6 +76,14 @@ endif()
...
+        DESTINATION ${PYTHON_BRIDGE_CPACK_PATH}
+        COMPONENT ${PYTHON_BRIDGE_CPACK_PATH})
```
Comment (slyubimt) [inline inference-engine/ie_bridges/python/CMakeLists.txt:83]:
```suggestion
        COMPONENT ${PYTHON_VERSION})
```

File: inference-engine/ie_bridges/python/CMakeLists.txt [hunk 3]
```diff
@@ -76,6 +76,14 @@ endif()
 
 # install
 
+ie_cpack_add_component(${PYTHON_BRIDGE_CPACK_PATH})
+
```
Comment (slyubimt) [inline inference-engine/ie_bridges/python/CMakeLists.txt:80]:
```suggestion
```

**Review by slyubimt — CHANGES_REQUESTED** [review]

File: inference-engine/ie_bridges/python/CMakeLists.txt [hunk 4]
```diff
@@ -76,6 +76,14 @@ endif()
...
+install(FILES requirements.txt
+        DESTINATION ${PYTHON_BRIDGE_CPACK_PATH}
+        COMPONENT ${PYTHON_BRIDGE_CPACK_PATH})
+
+ie_cpack(${PYTHON_BRIDGE_CPACK_PATH})
```
Comment (slyubimt) [inline inference-engine/ie_bridges/python/CMakeLists.txt:85]:
```suggestion
```

**Review by slyubimt — CHANGES_REQUESTED** [review]

**Review by slyubimt — APPROVED** [review]

**Pull request merged**